tox.ini

```diff
@@ -171,24 +171,22 @@ commands = coverage report -i
 # dependencies will generally be older than what is found in pip.
 
 # To obtain these versions, check the versions of these libraries
-# in the oldest support Ubuntu distro. These versions are from bionic.
+# in the oldest support Ubuntu distro. These versions are from noble.
```

Comment:
Why are we jumping to noble versions instead of jammy? Isn't jammy still within standard support until 2027-06-01 from /usr/share/distro-info/ubuntu.csv?